portfolio/src/main/webapp/style.css

```diff
@@ -1,25 +1,108 @@
 #content {
 	margin-left: auto;
 	margin-right: auto;
-	width: 650px;
+	width: 80vw;
 }
 
 #greeting-container {
-  margin-top: 20px;
+	color: white;
```

Comment:
Might be good to fix inconsistent indentation here. I'm guessing some lines were copied in with tab indents, and some lines were hand-written and the editor used space indents (happens all the time). Your editor may have an option to normalize indentation.